### PR TITLE
FIX: Plotting issues with brain images (bg_img) containing non-finite values

### LINKED ISSUE #1268: plot_stat_map does not display the bg anat when it contains NaNs
Presumably because thre sampling used in plotting does not handle Nans correctly.
This is suprising because I thought that this was not occurring in previous versions of Nilearn.
Maybe related to issue #1249 ?

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Attempts to FIX #1268 . Last time we fixed plotting issues with foreground images but not background image. This PR should hopefully fix this as well.

Please reviews ?

Thanks

### CODE REVIEW

File: nilearn/plotting/img_plotting.py [hunk 1]
```diff
@@ -639,6 +645,9 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
         A small threshold is applied by default to eliminate numerical
         background noise.
 
+        For visualization, non-finite values found in passed 'roi_img' or
+        'bg_img' are kept to zero.
```
Comment (GaelVaroquaux) [inline nilearn/plotting/img_plotting.py]: Phrasing : they are "set" to zero and not "kept" to zero

File: nilearn/plotting/img_plotting.py [hunk 2]
```diff
@@ -955,6 +964,9 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
         Arrays should be passed in numpy convention: (x, y, z)
         ordered.
 
+        For visualization, non-finite values found in passed 'stat_map_img' or
+        'bg_img' are kept to zero.
```
Comment (GaelVaroquaux) [inline nilearn/plotting/img_plotting.py]: Same comment

**Comment by bthirion** [comment]
All right, the issue is fixed as far as I am concerned. Thx !

**Comment by GaelVaroquaux** [comment]
LGTM. Merging. Thanks

**Pull request merged**